includes/class-plugin.php

```
@@ -14,51 +14,51 @@
-	/**
-	 * Plugin file
-	 *
-	 * @var string
-	 */
-	private $plugin_file;
+    /**
+     * Plugin file
+     *
+     * @var string
+     */
+    private $plugin_file;
```

PHPStan (2.0.3)

Property WPALLSTARS\PluginStarterTemplate\Plugin::$plugin_file is never read, only written.
See: https://phpstan.org/developing-extensions/always-read-written-properties

(property.onlyWritten)